app/assets/javascripts/discourse/app/components/emoji-picker.js

```diff
@@ -465,7 +465,7 @@ export default Component.extend({
 
   @bind
   handleOutsideClick(event) {
-    if (!document.querySelector(".emoji-picker")?.contains(event.target)) {
+    if (!event.target.closest(".emoji-picker")) {
```

Comment:
Used `closest` instead of `querySelector` + `.contains` so it matches the check in the `chat-emoji-picker`.